iceoryx_hoofs/test/moduletests/test_cxx_string.cpp

```diff
@@ -1286,22 +1323,24 @@ TYPED_TEST(stringTyped_test, CheckForEqualityWithEqualStringWithDifferentCapaWor
 {
     ::testing::Test::RecordProperty("TEST_ID", "a01cc09b-fe94-42db-bcbd-4f813f4acd62");
     using MyString = typename TestFixture::stringType;
-    constexpr auto STRINGCAP = MyString::capacity();
+    constexpr auto STRINGCAP = MyString::capacity() + 5U;
     this->testSubject = "M";
 
-    char testCharArray[STRINGCAP + 5U] = {'M'};
+    // NOLINTNEXTLINE(hicpp-avoid-c-arrays, cppcoreguidelines-avoid-c-arrays) cxx::string wraps char array
+    char testCharArray[STRINGCAP] = {'M'};
```

Comment:
This also feels like a regression. In previous tests `STRINGCAP` was related to the cxx::string and in this test it made it immediately clear that the char array and std::string had a larger capacity.